examples/lines.html

```diff
@@ -33,12 +33,23 @@
 }
 
 // instantiate our graph!
+console.log(seriesData)
+
+var d = new Date();
+d.setHours(d.getHours() + 1);
 
 var graph = new Rickshaw.Graph( {
 	element: document.getElementById("chart"),
 	width: 960,
 	height: 500,
 	renderer: 'line',
+	bands: [{
```

Comment:
nit: tabs

Reply:
We can update the jshint config in another PR. I would just leave as-is for now.